Content.Client/GameObjects/Components/HandheldLightComponent.cs

```diff
@@ -88,23 +88,32 @@ protected override void Update(FrameEventArgs args)
                 }
                 else
                 {
-                    level = 1 + (int) MathF.Round(charge * 6);
+                    if (charge > 0.8333) level = 6;
+                    else if (charge > 0.667) level = 5;
+                    else if (charge > 0.5) level = 4;
+                    else if (charge > 0.333) level = 3;
+                    else if (charge > 0.167) level = 2;
+                    else level = 1;
```

Comment:
What's wrong with using a stepper? Also there's a dedicated function for this.

Reply:
I don't know what a stepper or the dedicated function is, and the pre-existing code had two bugs in it so I just went with this.

Reply:
Yeah, I couldn't get this to work satisfactorily with `ContentHelpers.RoundToLevels`... one of the levels would always last longer than others and figuring out why was 2frustrating4me.

Reply:
Update to at least one of the helpers coz it do be ugly.